server/src/db/migrations/20220217164425_createSquids.cjs

```diff
@@ -11,7 +11,8 @@ exports.up = async (knex) => {
     table.string("name").notNullable();
     table.string("species").notNullable();
     table.string("experiencePoints").defaultTo(0).notNullable();
-    table.enu("specialPower", ["ink", "bioluminescence", "camouflage", null]);
+    table.enu("specialPower", ["ink", "camouflage", "bioluminescence", "flight", null]);
+    table.string("imageUrl");
     table.timestamp("createdAt").notNullable().defaultTo(knex.fn.now());
```

Comment:
it would be better practice to make a new migration file here, since your previous work has already been merged and if something goes wrong with the deployment of this feature to your app, you'd want to be able to roll back to the previous deployment's db state